src/Microsoft.DotNet.XHarness.TestRunners.Xunit/ThreadlessXunitTestRunner.cs

```diff
@@ -64,21 +64,21 @@ public override async Task Run(IEnumerable<TestAssemblyInfo> testAssemblies)
 
             if (EnvironmentVariables.IsLogTestStart())
             {
-                testSink.Execution.TestStartingEvent += args => { Console.WriteLine($"[STRT] {args.Message.Test.DisplayName}"); };
+                testSink.Execution.TestStartingEvent += args => { OnInfo($"[STRT] {args.Message.Test.DisplayName}"); };
             }
             testSink.Execution.TestPassedEvent += args =>
             {
-                Console.WriteLine($"[PASS] {EscapeNewLines(args.Message.Test.DisplayName)}");
+                OnDebug($"[PASS] {WasmXmlResultWriter.EscapeNewLines(args.Message.Test.DisplayName)}");
```

Comment:
WasmXmlResultWriter doesn't feel the right place for EscapeNewLines given how it is used.
Probably better to move it to https://github.com/dotnet/xharness/blob/main/src/Microsoft.DotNet.XHarness.Common/Utilities/StringUtils.cs

Reply:
The `Microsoft.DotNet.XHarness.Common` is referenced by CLI, but not by runner projects. Anyway I have added a new StringUtils class in `Microsoft.DotNet.XHarness.TestRunners.Common`